lib/config/options/index.ts

```diff
@@ -705,7 +705,9 @@ const options: RenovateOptions[] = [
     name: 'autodiscoverFilter',
     description: 'Filter the list of autodiscovered repositories.',
     stage: 'global',
-    type: 'string',
+    type: 'array',
+    subType: 'string',
+    allowString: true,
```

Comment:
@HonkingGoose This will automigrate to array for config files 😉

Reply:
Cool! 😄